rollup/mock_bridge/MockBridge.sol

```diff
@@ -312,4 +312,35 @@ contract MockBridgeL1 {
       mstore(memPtr, dataHash)
     }
   }
+
+  address private constant POINT_EVALUATION_PRECOMPILE_ADDRESS = 0x000000000000000000000000000000000000000A;
+  uint256 private constant BLS_MODULUS = 52435875175126190479447740508185965837690552500527637822603658699938581184513;
+
+  function verifyProof(
```

Comment:
What do we test with this method? Since this is not the same logic as the actual function that we'll add to `ScrollChain`

Reply:
It's only a smoke test with a contract call containing `blobhash` op, ensuring something wouldn't be broken in `eth_createAccessList`, `eth_estimateGas`. Previously, blob support of these JSON APIs in go SDK is still WIP, but it works fine now.

Mock interfaces for ScrollChain will be tested in `rollup-relayer`'s unit test.